### DISCUSSION TITLE
Wer verwendet die Funktion "Sende Datum und Uhrzeit an Gruppenadressen"?

### DISCUSSION THREAD

**Discussion by brutella** [brutella/hkknx-public#256] [Polls]
Aufgrund von Aufräumarbeiten würde ich gerne die Funktion *Sende Datum und Uhrzeit an Gruppenadressen* entfernen. Als Alternative würde ich ein Skript anbieten, was genau das selbe tut.

**Comment by Varone3000** [comment]
Sinnvoll für Leute ohne Zentrale oder IP Schnittstelle mit Uhr aber auf jeden Fall. Ich würde diese Funktion auf jeden Fall drin lassen.

**Reply by brutella** [reply]
Es sollte aber auch genügen, diese Funktion als vorgefertigtes Skript zur Verfügung zu stellen, oder?

**Reply by flotzi** [reply]
Würde die Funktion auch sehr gerne behalten. Habe sonst keine Möglichkeit sie auf den Bus zu senden und mit Scripts wollte ich nicht anfangen. Ist finde ich ein großer Vorteil von hkknx im Vergleich zu anderen knx bridges.

**Reply by Varone3000** [reply]
Ich kann auch keine Scriptsprache. Und mei. Alltag und Job lässt es auch nicht zu das noch zusätzlich zu lernen. Ich würde mich sogar freuen wenn man die Wettervorhersage aus Apple Wetter auf den Bus bringen könnte.

**Comment by brutella** [comment]
In Version [2.8.0](https://github.com/brutella/hkknx-public/releases/tag/2.8.0) gibt es jetzt das `ntp` Package. Mit diesem Package kann man die Zeit über einen NTP-Timeserver abfragen. Wenn man dann diese Zeit als `10.001`, `11.001` oder `19.001` kodiert und  an eine Gruppenadresse sendet, hat man die selbe Funktionalität wie bisher. Wie das geht, wir anhand eines [Beispiels im Online Handbuch](https://hochgatterer.me/de/hkknx/scripts/time/#ntp) erklärt.

**Comment by sirmoon137** [comment]
Ich habe versucht die funktion Zeit zu senden zu verwenden doch leider kommt am bus nur die sommerzeit an spich, mir wird aktuell mit date die richtige zeit am system und auch in HKKNX angezeigt aber am bus landet die Zeit mit -1h ... es verhält sich gleich wenn ich den NTP verwende. Wie kann man das ändern?

**Reply by sirmoon137** [reply]
mir ist gerade aufgefallen oben wird die GMT angezeig (Zeit -1h) unten aber die richtige. 
<img width="548" alt="Bildschirmfoto 2025-03-13 um 11 50 22" src="https://github.com/user-attachments/assets/258650f0-bc31-40dd-97d5-48cfeca2e3b7" />

**Reply by brutella** [reply]
Die Systemzeit zeigt die Zeit des Systems an. Dabei wird die Zeitzone berücksichtigt von dem Computer, auf dem hkknx läuft. Die Zeit unter "Datum & Uhrzeit" berücksichtigt die Zeitzone von deinem Browser.

Wenn die Zeitzonen unterschiedlich sind, ist die angezeigte Zeit auch unterschiedlich.

Je nachdem wo hkknx installiert ist, kannst du die Zeitzone selbst aktualisieren. 

(Das neue Raspberry Pi Image verwendet fix die Zeitzone `Europe/Vienna`. Es wäre sicher eine gute Idee, die Zeitzone über das Webinterface einstellbar zu machen. #371 )
